src/ORM/Connect/DBSchemaManager.php

```diff
@@ -412,7 +412,6 @@ public function requireTable(
                 if ($pos !== false) {
                     //If so, remove it and store that info separately
                     $arrayValue = substr($fieldSpec, $pos);
-                    $fieldSpec = substr($fieldSpec, 0, $pos);
```

Comment:
Removing this line keeps the options array in the field spec which is passed through into the injector. Without this the options _never_ get to the constructor.